interface.go

```diff
@@ -35,6 +35,11 @@ func (i InterfaceDesc) String() string {
 	return fmt.Sprintf("Interface %d (%d alternate settings)", i.Number, len(i.AltSettings))
 }
 
+type RawDescriptor struct {
```

Comment:
document the type. In particular whether the header of the descriptor (type+length) is a part of the data is an important information here.